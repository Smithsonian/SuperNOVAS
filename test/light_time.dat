
2450545.000000 SUN        S0 O0 A1:     0.000050

2450545.000000 SUN        S0 O1 A1:     0.005780

2450545.000000 SUN        S0 O2 A1:     0.599963

2451545.000000 SUN        S0 O0 A1:     0.000044

2451545.000000 SUN        S0 O1 A1:     0.005679

2451545.000000 SUN        S0 O2 A1:     0.606470

2452545.000000 SUN        S0 O0 A1:     0.000030

2452545.000000 SUN        S0 O1 A1:     0.005788

2452545.000000 SUN        S0 O2 A1:     0.611374
