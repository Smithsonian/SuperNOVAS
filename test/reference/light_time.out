
-10000.0 22+20      S2 O0 A0: ERROR 12 

-10000.0 22+20      S2 O1 A0: ERROR 12 

-10000.0 22+20      S2 O2 A0: ERROR 12 

-10000.0 16-20      S2 O0 A0: ERROR 12 

-10000.0 16-20      S2 O1 A0: ERROR 12 

-10000.0 16-20      S2 O2 A0: ERROR 12 

-10000.0 SUN        S0 O0 A0: ERROR 23 

-10000.0 SUN        S0 O1 A0: ERROR 23 

-10000.0 SUN        S0 O2 A0: ERROR 23 

-10000.0 EARTH      S0 O0 A0: ERROR 23 

-10000.0 EARTH      S0 O1 A0: ERROR 23 

-10000.0 EARTH      S0 O2 A0: ERROR 23 

     0.0 22+20      S2 O0 A0: ERROR 12 

     0.0 22+20      S2 O1 A0: ERROR 12 

     0.0 22+20      S2 O2 A0: ERROR 12 

     0.0 16-20      S2 O0 A0: ERROR 12 

     0.0 16-20      S2 O1 A0: ERROR 12 

     0.0 16-20      S2 O2 A0: ERROR 12 

     0.0 SUN        S0 O0 A0: ERROR 23 

     0.0 SUN        S0 O1 A0: ERROR 23 

     0.0 SUN        S0 O2 A0: ERROR 23 

     0.0 EARTH      S0 O0 A0: ERROR 23 

     0.0 EARTH      S0 O1 A0: ERROR 23 

     0.0 EARTH      S0 O2 A0: ERROR 23 

 10000.0 22+20      S2 O0 A0: ERROR 12 

 10000.0 22+20      S2 O1 A0: ERROR 12 

 10000.0 22+20      S2 O2 A0: ERROR 12 

 10000.0 16-20      S2 O0 A0: ERROR 12 

 10000.0 16-20      S2 O1 A0: ERROR 12 

 10000.0 16-20      S2 O2 A0: ERROR 12 

 10000.0 SUN        S0 O0 A0: ERROR 23 

 10000.0 SUN        S0 O1 A0: ERROR 23 

 10000.0 SUN        S0 O2 A0: ERROR 23 

 10000.0 EARTH      S0 O0 A0: ERROR 23 

 10000.0 EARTH      S0 O1 A0: ERROR 23 

 10000.0 EARTH      S0 O2 A0: ERROR 23 

 10000.0 22+20      S2 O0 A0: ERROR 12 

 10000.0 22+20      S2 O1 A0: ERROR 12 

 10000.0 22+20      S2 O2 A0: ERROR 12 

 10000.0 16-20      S2 O0 A0: ERROR 12 

 10000.0 16-20      S2 O1 A0: ERROR 12 

 10000.0 16-20      S2 O2 A0: ERROR 12 

 10000.0 SUN        S0 O0 A0: ERROR 23 

 10000.0 SUN        S0 O1 A0: ERROR 23 

 10000.0 SUN        S0 O2 A0: ERROR 23 

 10000.0 EARTH      S0 O0 A0: ERROR 23 

 10000.0 EARTH      S0 O1 A0: ERROR 23 

 10000.0 EARTH      S0 O2 A0: ERROR 23 

 10000.0 22+20      S2 O0 A0: ERROR 12 

 10000.0 22+20      S2 O1 A0: ERROR 12 

 10000.0 22+20      S2 O2 A0: ERROR 12 

 10000.0 16-20      S2 O0 A0: ERROR 12 

 10000.0 16-20      S2 O1 A0: ERROR 12 

 10000.0 16-20      S2 O2 A0: ERROR 12 

 10000.0 SUN        S0 O0 A0: ERROR 23 

 10000.0 SUN        S0 O1 A0: ERROR 23 

 10000.0 SUN        S0 O2 A0: ERROR 23 

 10000.0 EARTH      S0 O0 A0: ERROR 23 

 10000.0 EARTH      S0 O1 A0: ERROR 23 

 10000.0 EARTH      S0 O2 A0: ERROR 23 

-10000.0 22+20      S2 O0 A1: ERROR 12 

-10000.0 22+20      S2 O1 A1: ERROR 12 

-10000.0 22+20      S2 O2 A1: ERROR 12 

-10000.0 16-20      S2 O0 A1: ERROR 12 

-10000.0 16-20      S2 O1 A1: ERROR 12 

-10000.0 16-20      S2 O2 A1: ERROR 12 

-10000.0 SUN        S0 O0 A1:     0.000024

-10000.0 SUN        S0 O1 A1:     0.005848

-10000.0 SUN        S0 O2 A1:     0.609167

-10000.0 EARTH      S0 O0 A1:     0.005834

-10000.0 EARTH      S0 O1 A1:     0.000000

-10000.0 EARTH      S0 O2 A1:     0.605790

     0.0 22+20      S2 O0 A1: ERROR 12 

     0.0 22+20      S2 O1 A1: ERROR 12 

     0.0 22+20      S2 O2 A1: ERROR 12 

     0.0 16-20      S2 O0 A1: ERROR 12 

     0.0 16-20      S2 O1 A1: ERROR 12 

     0.0 16-20      S2 O2 A1: ERROR 12 

     0.0 SUN        S0 O0 A1:     0.000044

     0.0 SUN        S0 O1 A1:     0.005679

     0.0 SUN        S0 O2 A1:     0.606470

     0.0 EARTH      S0 O0 A1:     0.005671

     0.0 EARTH      S0 O1 A1:     0.000000

     0.0 EARTH      S0 O2 A1:     0.605681

 10000.0 22+20      S2 O0 A1: ERROR 12 

 10000.0 22+20      S2 O1 A1: ERROR 12 

 10000.0 22+20      S2 O2 A1: ERROR 12 

 10000.0 16-20      S2 O0 A1: ERROR 12 

 10000.0 16-20      S2 O1 A1: ERROR 12 

 10000.0 16-20      S2 O2 A1: ERROR 12 

 10000.0 SUN        S0 O0 A1:     0.000025

 10000.0 SUN        S0 O1 A1:     0.005842

 10000.0 SUN        S0 O2 A1:     0.601308

 10000.0 EARTH      S0 O0 A1:     0.005864

 10000.0 EARTH      S0 O1 A1:     0.000000

 10000.0 EARTH      S0 O2 A1:     0.605779

 10000.0 22+20      S2 O0 A1: ERROR 12 

 10000.0 22+20      S2 O1 A1: ERROR 12 

 10000.0 22+20      S2 O2 A1: ERROR 12 

 10000.0 16-20      S2 O0 A1: ERROR 12 

 10000.0 16-20      S2 O1 A1: ERROR 12 

 10000.0 16-20      S2 O2 A1: ERROR 12 

 10000.0 SUN        S0 O0 A1:     0.000025

 10000.0 SUN        S0 O1 A1:     0.005842

 10000.0 SUN        S0 O2 A1:     0.601308

 10000.0 EARTH      S0 O0 A1:     0.005864

 10000.0 EARTH      S0 O1 A1:     0.000000

 10000.0 EARTH      S0 O2 A1:     0.605779

 10000.0 22+20      S2 O0 A1: ERROR 12 

 10000.0 22+20      S2 O1 A1: ERROR 12 

 10000.0 22+20      S2 O2 A1: ERROR 12 

 10000.0 16-20      S2 O0 A1: ERROR 12 

 10000.0 16-20      S2 O1 A1: ERROR 12 

 10000.0 16-20      S2 O2 A1: ERROR 12 

 10000.0 SUN        S0 O0 A1:     0.000025

 10000.0 SUN        S0 O1 A1:     0.005842

 10000.0 SUN        S0 O2 A1:     0.601309

 10000.0 EARTH      S0 O0 A1:     0.005864

 10000.0 EARTH      S0 O1 A1:     0.000000

 10000.0 EARTH      S0 O2 A1:     0.605779
